scripts/jenkins/motr-hare-1node-cluster-rocky-linux.groovy

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env groovy
+pipeline { 
+    agent {
+        node {
+           label 'ssc-vm-g3-rhev4-1327'
+        }
+    }
+	
+    parameters {
+        string(name: 'NODE_HOST', defaultValue: '', description: 'Node 1 Host FQDN',  trim: true)
+        string(name: 'NODE_USER', defaultValue: '', description: 'Host machine root user',  trim: true)
+        string(name: 'NODE_PASS', defaultValue: '', description: 'Host machine root user password',  trim: true)
+    }
+
+    options {
+        timeout(time: 180, unit: 'MINUTES')
+        timestamps()
+        ansiColor('xterm') 
+        buildDiscarder(logRotator(numToKeepStr: "30"))
+    }
+
+    environment {
+        NODE_PASS = "${NODE_PASS.isEmpty() ? NODE_DEFAULT_SSH_CRED_PSW : NODE_PASS}"
+    }
+
+    stages {
+        stage('Checkout') {
+            steps {
+                checkout([$class: 'GitSCM', branches: [[name: "EOS-28455"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'PathRestriction', excludedRegions: '', includedRegions: 'scripts/third-party-rpm/.*']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: "https://github.com/rkothiya/cortx-motr"]]])
+            }
+        }
+        stage ('Build rpm packages') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                sh label: 'to build motr and hare rpm', script: '''
+                    sshpass -p ${NODE_PASS} ssh -o StrictHostKeyChecking=no ${NODE_USER}@${NODE_HOST} hostname
+                    sshpass -p ${NODE_PASS} ssh -o StrictHostKeyChecking=no ${NODE_USER}@${NODE_HOST} git clone https://github.com/rkothiya/cortx-motr
```

Comment:
https://github.com/rkothiya/cortx-motr

Use the main repo, this job needs to triggered for each PR as a pre-merge job.

Reply:
Sure, I had used my repo for testing as the main repo does not contain this script. Now that I have tested it. I have reverted to the main repo.